src/appforms/src/core/040-Model06Submission.js

```diff
@@ -816,21 +820,30 @@ appForm.models = function(module) {
       }
     }
   };
+
   Submission.prototype.removeFileValue = function(hashName){
     var subFiles = this.get('filesInSubmission', []);
     if(typeof(hashName) === "string" && subFiles.indexOf(hashName) > -1){
       subFiles.splice(subFiles.indexOf(hashName),1);
       this.set('filesInSubmission', subFiles);
     }
   };
-  Submission.prototype.getInputValueByFieldId = function(fieldId, cb) {
+
+  Submission.prototype.getInputValueByFieldId = function(fieldId, sectionIndex, cb) {
```

Comment:
diff doest load due to size.
https://github.com/witmicko/fh-js-sdk/blob/9adae83b91e0d0a58cfc7d0d069c73486faa895b/src/appforms/tests/tests/core/040-Model06Submission.js#L1020